code/game/objects/items/food/_food.dm

```diff
@@ -51,13 +51,13 @@
 	///Buff given when a hand-crafted version of this item is consumed. Randomized according to crafting_complexity if not assigned.
 	var/datum/status_effect/food/crafted_food_buff = null
 
-/obj/item/food/New(loc, starting_reagent_purity, no_base_reagents = FALSE, ...)
+/obj/item/food/Initialize(mapload, starting_reagent_purity, no_base_reagents = FALSE)
+	. = ..()
```

Comment:
```suggestion
```
(Called parent twice)
I assume it was used in `New` to beat the parent call of Init, to set up the food reagents list